src/sass/partials/base/_base.scss

```diff
@@ -0,0 +1,312 @@
+
+._wrapper {
+  @include wrapper;
+}
+
+.base {
+  &--a {
+    border-bottom: 2px solid setting-get('primary color');
+    color: setting-get('primary color');
+    text-decoration: none;
+
+    &:hover {
+      border-color: setting-get('secondary color');
+      color: setting-get('secondary color');
+      transition: setting-get('fade');
+    }
+  }
+
+  // Major Third Type Scale
+
+  &--h1 {
+    font-size: 2.441em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h2 {
+    font-size: 1.953em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h3 {
+    font-size: 1.563em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h4 {
+    font-size: 1.25em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h5 {
+    font-size: 1em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h6 {
+    font-size: .8em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--p {
+    font-weight: setting-get('normal font weight');
+    @include breakpoint(setting-get('max paragraph length')) {
```

Comment:
I've got a concern with this styling; specifically what happens if a paragraph lives inside a section whose length is larger than 40em?

Also, `max paragraph length` is `800px` which is larger than `40em` at at `16px` base font size. Does this mean that the intention is to allow paragraph tags to grow to `800px` then snap back to `640px` (`16px/em * 40em`)? Is this true for all of the things using `max paragraph length`? If the intention is to never have these items grow larger than `40em`, we should just place the `max-width` on them w/o the breakpoint (and probably set `max paragraph length` to that size)